plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -1212,6 +1215,8 @@ public void verifySupportsJoinPushdownWithFullJoinDeclaration()
     @Test
     public void testJoinPushdown()
     {
+        skipTestUnless(hasBehavior(SUPPORTS_CREATE_TABLE_WITH_DATA));
```

Comment:
This is a bad sign in this PR. `SUPPORTS_JOIN_PUSHDOWN` is true & `SUPPORTS_CREATE_TABLE_WITH_DATA` is false in TestExasolConnectorTest. Thus, this method will be skipped silently.

Reply:
Correct.

There is a comment below: `If a connector supports Join pushdown, but does not allow CTAS, we need to make the table creation here overridable.`. I don't know the framework well enough to implement this.

Would it be an option for you to remove SUPPORTS_JOIN_PUSHDOWN?

Reply:
I would recommend removing support for JOIN pushdown from this PR.